packages/flat-stores/src/whiteboard-store/index.ts

```diff
@@ -115,7 +115,7 @@ export class WhiteboardStore {
     };
 
     public updateWritable = (isWritable: boolean): void => {
-        this.isWritable = isWritable;
+        this.isWritable = !!isWritable;
```

Comment:
Hi, this line looks a little strange, the boolean type variable `isWritable` needs to be executed `!!` operator. Is it useful?

Reply:
The issue is that it did in fact receive an `undefined` value and whiteboard sdk crashed with that input, which is called in classroom-store's `whiteboardStore.updateWritable()` codes. It is just easier to fix that issue here.

I can refactor the code to fix the real causes.

Reply:
> The issue is that it did in fact receive an `undefined` value and whiteboard sdk crashed with that input, which is called in classroom-store's `whiteboardStore.updateWritable()` codes. It is just easier to fix that issue here.
> 
> I can refactor the code to fix the real causes.

Hahaha, I got it.